media-sound/yoshimi/yoshimi-1.7.1-r1.ebuild

```diff
@@ -11,7 +11,7 @@ SRC_URI="https://github.com/${PN^}/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="amd64 x86"
+KEYWORDS="~amd64 ~x86"
```

Comment:
If you edit KEYWORDS then do not revbump, we need to leave a stable version in tree.

Reply:
Thanks